rust/psibase/src/main.rs

```diff
@@ -87,6 +87,11 @@ enum Command {
         package_source: Vec<String>,
 
         services: Vec<OsString>,
+
+        /// Configure compression level for boot package file uploads
+        /// (1=fastest, 11=most compression)
+        #[clap(short = 'z', long, value_name = "LEVEL", default_value = "4")]
```

Comment:
I think the syntax should be a number of z's equal to the compression level. i.e. `-zzz`, or `-zzzzzzzzzzz` for the most compression.

Reply:
Why? That seems odd/unconventional and less user-friendly